src/google/protobuf/compiler/objectivec/BUILD.bazel

```diff
@@ -6,6 +6,15 @@ load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
 load("//build_defs:cpp_opts.bzl", "COPTS")
 
+cc_library(
+    name = "names",
+    hdrs = ["names.h"],
+    copts = COPTS,
+    include_prefix = "google/protobuf/compiler/objectivec",
```

Comment:
why is this needed?

Reply:
This is a better practice for modeling visibility.  Bazel doesn't enforce "private" internal headers today

Reply:
`includes` should get propagated by `cc_library`, so it impacts everything that depends on it.  If you only want it to apply to this target, a `copt` would be the better choice. But I don't think it's actually needed since the compiler should always check in the same directory as the source file doing the `#include`.